packages/plugins/plugin-navtree/src/components/NavTree/NavTree.stories.tsx

```diff
@@ -36,13 +38,54 @@ faker.seed(1234);
 
 const StoryState = defineCapability<{ tab: string }>('story-state');
 
+// TODO(burdon): How to adjust existing surfaces for this test?
+// TODO(burdon): How to toggle attention in this test?
+// TODO(burdon): Fix outline (e.g., button in sidebar nav is clipped when focused).
+// TODO(burdon): Consider similar containment of: Table, Sheet, Kanban Column, Form, etc.
+
+const container = 'flex flex-col grow gap-2 p-4 rounded-md';
+
+const TestPanel = () => {
+  // TODO(burdon): Where should attention be applied?
```

Comment:
Anything can have attendable attributes, which is different from having an attention background or applying other attention-related signifiers (e.g. toolbars becoming partially opaque).

The attributes are for annotating the object id of which the element is rendering a representation, they don’t apply any styles.

Reply:
again, please just update to make this story a canonical example.
noting that sheet, table, and kanban all do this inconsistently (hence the bug in surface attention we discussed), so again i want to make this story definitive.